src/rdf4cpp/rdf/graph/node_manager/BNodeBackend.h

```diff
@@ -0,0 +1,23 @@
+#ifndef RDF4CPP_BNODEBACKEND_H
+#define RDF4CPP_BNODEBACKEND_H
+
+#include <rdf4cpp/rdf/graph/node_manager/NodeID.h>
+
+#include <compare>
+#include <string>
+
+namespace rdf4cpp::rdf::graph::node_manager {
+
+class BNodeBackend {
+    std::string identifier_;
```

Comment:
why identifier_ and not identifier ?

Reply:
to avoid a name clash with https://github.com/rdf4cpp/rdf4cpp/blob/1f8c0ca63f063a87d6a032c33274bf1a7c2913ae/src/rdf4cpp/rdf/graph/node_manager/BNodeBackend.h#L19

I find suffix `_` preferable over prefix, because IDE auto completion works better this way.